### ISSUE TITLE
redis slave should not run addslots

### ISSUE THREAD

**Issue by noelmcloughlin** [noelmcloughlin/k8s-statefulsets#1]
I think line 414 should be deleted. I added during troubleshooting stuff.

https://github.com/noelmcloughlin/k8s-statefulsets/blob/cc7289404b0335b85d19b75d0928411f3fd5b6f8/redis-set.yaml#L414

```
    if [ "$ORDINAL" -eq "0" ]; then
        ## Controllers
        /usr/local/bin/redis-cli cluster addslots "$(seq 0 16383)"
    else
        /usr/local/bin/redis-cli cluster addslots "$(seq 0 16383)"
        # Slaves of a random master
```

**Comment by noelmcloughlin** [comment]
Fixed by 64f9247

**Comment by noelmcloughlin** [comment]
Also removed enclosing double quotes in commit: 2d648804cefd4300efb062be8616228c9dfea652
